src/ose.js

```diff
@@ -16,6 +16,7 @@ import * as party from "./module/party";
 import { OseCombat } from "./module/combat";
 import * as renderList from "./module/renderList";
 import { OsePartySheet } from "./module/party/party-sheet";
+import "./ose.scss";
```

Comment:
What's this do if we're already pulling our styles in from `system.json`?

Reply:
This is included to convert `scss` to `css`. The rollup plugin for `scss` captures that bundle and emits it to `dist/` in the location that `system.json` expects it.

Reply:
seems appropriate, it fits the rollup-plugin-scss usage example
<img width="736" alt="image" src="https://user-images.githubusercontent.com/1410433/195435211-8b395a3c-856c-459a-99b3-5bdc20fcd889.png">